src/content/en/2019/compression.md

```diff
@@ -131,61 +131,68 @@ Appoximately 38% of HTTP responses are delivered with text based compression. Th
 
-In the graph below, the top 25 content types are displayed with box sizes representing the relative amount of requests. The color of each box represents how many of these resources were served compressed. Most of the media content is shaded red, which is expected since gzip and brotli would have little to no benefit for them.  Most of the text content is shaded green to indicate that they are being compressed. However, the light green shading for some content types indicate that they are not compressed as consistently as the others. 
+In the graph below, the top 25 content types are displayed with box sizes representing the relative number of requests. The color of each box represents how many of these resources were served compressed. Most of the media content is shaded red, which is expected since gzip and brotli would have little to no benefit for them.  Most of the text content is shaded green to indicate that they are being compressed. However, the light green shading for some content types indicate that they are not compressed as consistently as the others. 
 
-![](../../../static/images/2019/15_Compression/ch15_fig2_compressed_content_types.jpg)
+![Top 25 Compressed Content Types](../../../static/images/2019/15_Compression/ch15_fig2_compressed_content_types.jpg)
 
-Filtering out the 6 most popular content types allows us to see the rest of these content types more clearly.The application/json and image/svg+xml content types are compressed less than 65% of the time.   
+Filtering out the 7 most popular content types allows us to see the rest of these content types more clearly.
 
-Most of the custom web fonts are served without compression, since they are already in a compressed format. However font/ttf is compressible, but only 84% of TTF font requests are being served with compression.
+![Filtered Top Compressed Content Types](../../../static/images/2019/15_Compression/ch15_fig3_compressed_content_types_filtered.jpg)
 
-![](../../../static/images/2019/15_Compression/ch15_fig3_compressed_content_types_filtered.jpg)
+The application/json and image/svg+xml content types are compressed less than 65% of the time.   
 
-The graphs below illustrates the breakdown of compression techniques used for each content type. Looking at the top 3 content types, we can see that across both Desktop and Mobile there are major gaps in compressing some of the most frequently requested content types. 56% of text/html as well as 18% of application/javascript and text/css resources are not being compressed. This presents a significant performance opportunity.. 
+Most of the custom web fonts are served without compression, since they are already in a compressed format. However, font/ttf is compressible, but only 84% of TTF font requests are being served with compression so there is still room for improvement here.
```

Comment:
Code format
```suggestion
Most of the custom web fonts are served without compression, since they are already in a compressed format. However, `font/ttf` is compressible, but only 84% of TTF font requests are being served with compression so there is still room for improvement here.
```